lib/Handlers/code/utils.ts

```diff
@@ -24,7 +29,7 @@ export const vmExecute = (data: { code: string; variables: Record<string, any> }
             });
           })();`;
 
-  vm.run((safe && clearContext) + data.code);
+  vm.run(`${safe ? clearContext : ''} ${data.code}`);
```

Comment:
existing bug

Reply:
curious about what the `clearContext` bug causes?

```
(function() {
            Function = undefined;
            const keys = Object.getOwnPropertyNames(this).concat(['constructor']);
            keys.forEach((key) => {
              const item = this[key];
              if (!item || typeof item.constructor !== 'function') return;
              this[key].constructor = undefined;
            });
          })()
```

Reply:
works fine for `safe == true`, if `safe == false` the resulting string is `falsedata_code_content`